packages/web-components/src/utils/utils.ts

```diff
@@ -30,9 +30,11 @@ export function getSlottedNodes(
       (node: any) => node.tagName === 'SLOT',
     ).length > 0;
 
-  // This will only get the first slot on a component
+  // This will check all the slots on a component
   const children = hasShadowDOM
-    ? root.shadowRoot.querySelector('slot').assignedNodes()
+    ? Array.from(root.shadowRoot.querySelectorAll('slot')).map(
```

Comment:
Why are these changes happening?

Reply:
The `va-radio` already includes a slot for the `va-radio-option`. When I added the "hint" slot, this util would only return the first (hint) slot, so all the `va-radio-options` did not get returned and thus the arrow keys to move between options did not work.

This util was updated to find _all slots_ and search through them for the matching `nodeName`

Reply:
Fixed